recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python
+    - python >=3.6
```

Comment:
Where is this >=3.6 requirement coming from? The [upstream setup.py](https://github.com/mpld3/mpld3/blob/v0.5.7/setup.py#L58-L61) suggests it support Python 2.6, 2.7, 3.3, and 3.4.

Also, this is not a noarch package, so `python >=something` should not be in the `run` requirements.

Reply:
Conda-forge built it as **noarch** https://github.com/conda-forge/mpld3-feedstock/blob/ec26951c51d2b1a9a45c7f8703ba422add1c49ce/recipe/meta.yaml#L13 fron v0.3 https://github.com/conda-forge/mpld3-feedstock/commit/d94be77aabb1a92ef76259d750bca05e2bce8229#
Should we build it also as noarch?

Reply:
Currently the package is a noarch on conda-forge, do we want to have the package reflect the upstream? or do we want to keep it as it was before?